tests/modules/programs/browserpass/browserpass.nix

```diff
@@ -6,20 +6,23 @@ with lib;
   config = {
     programs.browserpass = {
       enable = true;
-      browsers = [ "chrome" "chromium" "firefox" "vivaldi" ];
+      browsers = [ "brave" "chrome" "chromium" "firefox" "vivaldi" ];
     };
 
     nmt.script = if pkgs.stdenv.hostPlatform.isDarwin then ''
-      for dir in "Google/Chrome" "Chromium" "Mozilla" "Vivaldi"; do
+      for dir in "BraveSoftware/Brave-Browser" "Google/Chrome" "Chromium" "Mozilla" "Vivaldi"; do
         assertFileExists "home-files/Library/Application Support/$dir/NativeMessagingHosts/com.github.browserpass.native.json"
       done
 
       for dir in "Google/Chrome" "Chromium" "Vivaldi"; do
         assertFileExists "home-files/Library/Application Support/$dir/policies/managed/com.github.browserpass.native.json"
       done
     '' else ''
-      for dir in "google-chrome" "chromium" "vivaldi"; do
+      for dir in "BraveSoftware/Brave-Browser" "google-chrome" "chromium" "vivaldi"; do
         assertFileExists "home-files/.config/$dir/NativeMessagingHosts/com.github.browserpass.native.json"
+      done
+
+      for dir in "google-chrome" "chromium" "vivaldi"; do
```

Comment:
Isn't `"BraveSoftware/Brave-Browser"` missing here?

Reply:
I just checked and it seems like brave is reading policies just from `/etc/brave/policies` on linux (https://github.com/brave/brave-browser/issues/19052)

Reply:
So it might be worth adding a comment about it.